src/braket/analog_hamiltonian_simulator/rydberg/validators/atom_arrangement.py

```diff
@@ -13,7 +14,17 @@ class AtomArrangementValidator(AtomArrangement):
 
     # Each site has two coordinates (minItems=maxItems=2)
     @root_validator(pre=True, skip_on_failure=True)
-    def sites_have_length_2(cls, values):
+    def sites_have_length_2(cls, values: Dict) -> Dict:
+        """
+        Validate that the sites in the atom arrangement have only two coordinates
+
+        Args:
+            values (Dict): The site and capability constants
+
+        Returns:
+            Dict: The validated sites
```

Comment:
It's unclear why you need to return anything if sites that don't fit the dimensions will throw an exception. Can you update the documentation or change the function to reflect if/why you need both?

Reply:
Honestly I am not sure why the function is written in this way as I am less with familiar with pydantic. What is the best practice to write documentation for a root_validator?